### PR TITLE
[fix bug 1283877] Mozilla newsletter landing page

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Creates a new landing page for the Mozilla newsletter so we have a single destination to point to. Features some new strings and I believe we do want to localize this page but l10n isn't a blocker.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1283877

## Testing
Just the usual. Make sure the page renders well in A-level browsers and degrades well in others, make sure the form works as expected.

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by craigcook** [comment]
Pending a final round of stakeholder review so marking WIP/DNM for the moment.

File: bedrock/newsletter/urls.py [hunk 1]
```diff
@@ -46,4 +47,6 @@
     url('^newsletter/$',
         views.newsletter_subscribe,
         name='newsletter.subscribe'),
+
+    page('^newsletter/mozilla', 'newsletter/mozilla.html'),
```
Comment (jpetto) [inline bedrock/newsletter/urls.py]: Don't think we need the `^` when using the `page` helper.

File: media/css/newsletter/newsletter-mozilla.less [hunk 2]
```diff
@@ -0,0 +1,225 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import (reference) '../pebbles/lib.less';
```
Comment (jpetto) [inline media/css/newsletter/newsletter-mozilla.less]: Ooh, I did not know about `(reference)`. Neat!

**Comment by jpetto** [comment]
Doesn't look like `intro-mozsummit-4000.jpg` is used anywhere. Can remove?

**Comment by jpetto** [comment]
screen shot 2016-07-08 at 3 50 31 pm

Form fields bleed into the margin at 320px wide.

**Comment by jpetto** [comment]
screen shot 2016-07-08 at 3 55 57 pm

Ah, thank you message bleeds a bit as well.

File: media/css/pebbles/components/footer.less [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            li {
+                float: none;
+            }
+        }
+
+        li {
+            float: left;
+
+            &.wrap:before {
+                content: "\00A0\00B7\00A0"; /* &nbsp;&middot;&nbsp; */
```
Comment (jpetto) [inline media/css/pebbles/components/footer.less]: @alexgibson told me to use single quotes for `content`. 😁

File: media/js/newsletter/mozilla.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function () {
+    'use strict';
+
+    $(document).ajaxSuccess(function(evt, xhr, settings, response) {
```
Comment (jpetto) [inline media/js/newsletter/mozilla.js]: Just to make things really obvious, add a comment here explaining what this is doing?

File: media/css/newsletter/newsletter-mozilla.less (hunk 2 above)
Comment (craigcook) [inline media/css/newsletter/newsletter-mozilla.less]: I just learned about it too! Gonna start using it everywhere in the refactor.

**Comment by craigcook** [comment]
I had an explicit `width: @width-phone;` on the form but that's too wide when it runs over the padding on the container. Easy fix.

File: media/css/pebbles/components/footer.less (hunk 3 above)
Comment (craigcook) [inline media/css/pebbles/components/footer.less]: This was copied from sandstone, which predates the styleguide. I'll correct it 😊

**Comment by jpetto** [comment]
A few small nits, but overall looks good. Tested in Fx, Chrome, & IE 9 and all work/look as expected.

r+wc 🌞

**Pull request merged**